nailgun/entities.py

```diff
@@ -1807,6 +1807,25 @@ def path(self, which=None):
             )
         return super(ProvisioningTemplate, self).path(which)
 
+    def read(self, entity=None, attrs=None, ignore=None, params=None):
+        """Provide a default value for ``entity``.
+
+        By default, ``nailgun.entity_mixins.EntityReadMixin.read`` provides a
+        default value for ``entity`` like so::
+
+            entity = type(self)()
+
+        However, :class:`ProvisioningTemplate` requires that a ``audit_comment`` is
+        provided, so this technique will not work.
+        """
+        # read() should not change the state of the object it's called on, but
+        # super() alters the attributes of any entity passed in. Creating a new
+        # object and passing it to super() lets this one avoid changing state.
```

Comment:
Where are you creating a new object? Is this comment correct?

Reply:
It's not my original comment (i copy&pasted it from different entity where the exact thing is happening), but the object it refers to is created here:

https://github.com/SatelliteQE/nailgun/pull/723/files#diff-3822710a997c3f2f7c3e93e884af69caR1827